index.html

```diff
@@ -2274,67 +2275,78 @@ <h2>
+If the technology involves authentication, particularly user-host
+authentication, the security of the authentication method MUST be clearly
+specified.
+            </li>
+          </ul>
+        </li>
 
-      <ol start="3">
         <li>
-This section MUST also discuss, per Section 5 of [[RFC3552]],
-residual risks (such as the risks from compromise in a related
-protocol, incorrect implementation, or cipher) after threat
-mitigation has been deployed.
+This section MUST discuss, per Section 5 of [[RFC3552]], residual risks (such as
+the risks from compromise in a related protocol, incorrect implementation, or
+cipher) after threat mitigation was deployed.
         </li>
 
         <li>
-This section MUST provide integrity protection and update
-authentication for all operations required by
-Section <a href="#did-operations"></a>.
+This section MUST provide integrity protection and update authentication for all
+operations required by Section <a href="#did-operations"></a>.
         </li>
 
         <li>
-Where <a>DID methods</a> make use of peer-to-peer computing resources
-(such as with all known <a>DLTs</a>), the expected burdens of those
-resources SHOULD be discussed in relation to denial of service.
+Where <a>DID methods</a> make use of peer-to-peer computing resources, such as
+with all known <a>DLTs</a>, the expected burdens of those resources SHOULD be
+discussed in relation to denial of service.
         </li>
 
         <li>
 Method-specific endpoint authentication MUST be discussed. Where
-<a>DID methods</a> make use of <a>DLTs</a> with varying network topology, sometimes
-offered as "light node" or "<a href="https://en.bitcoin.it/wiki/Thin_Client_Security">thin client</a>"
-implementations to reduce required computing resources, the security
-assumptions of the topology available to implementations of the <a>DID
-method</a> MUST be discussed.
+<a>DID methods</a> make use of <a>DLTs</a> with varying network topology,
+sometimes offered as <em>light node</em> or
+<em><a href="https://en.bitcoin.it/wiki/Thin_Client_Security">thin client</a></em>
+implementations to reduce required computing resources, the security assumptions
+of the topology available to implementations of the <a>DID method</a> MUST be
+discussed.
         </li>
 
         <li>
 <a>DID methods</a> MUST discuss the policy mechanism by which <a>DIDs</a> are
 proven to be uniquely assigned. A <a>DID</a> fits the functional definition
-of a URN as defined in [[RFC8141]] &mdash; a persistent identifier that is
-assigned once to a resource and never reassigned. In a security
-context this is particularly important since a <a>DID</a> may be used to
-identify a specific party subject to a specific set of authorization
-rights.
+of a URN, as defined in [[RFC8141]]. That is, a persistent identifier that is
+assigned once to a resource and never reassigned. In a security context this is
+particularly important because a <a>DID</a> might be used to identify a specific
```

Comment:
```
in [[RFC8141]]. That is, a persistent identifier that is
assigned once to a resource and never reassigned. In a security context this is
particularly important because
```
->
```
in [[RFC8141]]; in other words, a <a>DID</a> is a persistent
identifier that is assigned once to a resource and never reassigned to a
different resource. This is particularly important in a security context,
because
```

Reply:
The first sentence is a statement that can stand all by itself. I would prefer to have the extra explanation as a separate sentence. I did incorporate your other changes, however.

`A <a>DID</a> fits the functional definition
of a URN, as defined in [[RFC8141]]. That is, a <a>DID</a> is a persistent
identifier that is assigned once to a resource and never reassigned to a
different resource. This is particularly important in a security context because
a <a>DID</a> might be used to identify a specific party subject to a specific
set of authorization rights.`